.github/dependabot.yml

```diff
@@ -63,3 +63,13 @@ updates:
       time: "23:30"
       timezone: Europe/Paris
     open-pull-requests-limit: 2
+
+  - package-ecosystem: npm
+    directory: /oxidation/client/electron
+    pull-request-branch-name:
+      separator: "-"
+    schedule:
+      interval: monthly
+      time: "23:30"
+      timezone: Europe/Paris
+    open-pull-requests-limit: 2
```

Comment:
Don't we also lack `binding/neon` & `binding/web` ?

- `binding/neon` have a `package-lock.json`
- `binding/web` have a `yarn.lock` but empty